internal/tofu/eval_import.go

```diff
@@ -28,10 +27,6 @@ func evaluateImportIdExpression(expr hcl.Expression, ctx EvalContext) (string, t
 		})
 	}
 
-	// The import expression is declared within the root module
-	// We need to explicitly use that context
-	ctx = ctx.WithPath(addrs.RootModuleInstance)
-
```

Comment:
Would you mind explaining why this was removed here? It's a bit unclear

Reply:
Yes, of course.
We moved it to the function that is calling `evaluateImportIdExpression()` [here](https://github.com/opentofu/opentofu/pull/1270/files#diff-b57ebc9c33d78d38d14349f826cce5d6b06348b1ce868061075d79c03e827d57R121). So now we get this value in the parameter `ctx EvalContext`.